tsqc/pages/mutations.py

```diff
@@ -102,6 +102,41 @@ def make_muts_panel(log_y, tsm):
         annot_track = make_annotation_plot(tsm, genes_df)
         layout += annot_track
 
+    selection_stream = hv.streams.Selection1D(source=points)
+    pops = ["Pop_A", "Pop_B", "Pop_C"]
+
+    def update_pop_freq_plot(index):
+        if not index:
+            return hv.Bars([], kdims="pop", vdims="freq").opts(
+                width=config.PLOT_WIDTH, height=400, title="Tap on a mutation"
+            )
+        mut_data = muts_df.loc[index[0]]
+        freqs = mut_data[pops].values
+        bars = hv.Bars(zip(pops, freqs), "pop", "freq").opts(
+            width=config.PLOT_WIDTH,
+            height=400,
+            framewise=True,
+            title=f"Mutation {index[0]}",
+            ylim=(0, 20),
+        )
+        return bars
+
+    def update_mut_info_table(index):
+        if not index:
+            return hv.Table([], kdims=["info"], vdims=["value"]).opts(
+                width=config.PLOT_WIDTH, title="Tap on a mutation"
+            )
+        mut_data = muts_df.loc[index[0]].drop(pops)
```

Comment:
I'm not sure how to avoid accessing the df row twice. I'm already doing the same thing in `update_pop_freq_plot`